content/blog/2024-07-31-tmix-07.md

```diff
@@ -0,0 +1,64 @@
++++
+title = "This Month in Xilem, July 2024"
+authors = ["Daniel McNab", "Olivier Faure"]
++++
+
+This month saw contributions on Linebender projects from volunteer contributors.
+Philipp Mildenberger especially is a major driver for progress on Xilem and especially the `xilem_web` crate.
+
+## Xilem
+
+- [xilem#423][] integrates Tokio into Xilem, showing how the Xilem model can interact with async work.
+- [xilem#428][] by Philipp adds explicit support for Flex parameters.
+- [xilem#440][], [xilem#448], etc add support Async support and experiments in Xilem Web
+
+## Masonry
+
+The most impactful change on the Masonry side was implementing the ["Widgets in arenas" RFC](https://github.com/linebender/rfcs/blob/main/rfcs/0006-widgets-in-arenas.md) in [xilem#396][].
+Widgets are now stored in a global arena, which has little impact on the public APIs but make future changes a lot easier to implement.
+
+- [xilem#417][] exposes an unstable API for embedding Masonry and Xilem into an existing Winit application.
+- [xilem#418][], [xilem#421][] improved our unofficial support for iOS.
+- [rfcs#7][] specifies a clearer architecture for passes, including simplifications necessary for compositor use.
+  Actually integrating with the compositor is still out of scope [for reasons Raph has laid out before](https://raphlinus.github.io/ui/graphics/2020/09/13/compositor-is-evil.html), but the new pass specification gives us a starting point when we get there.
+
+## Vello
+
+- The [GPU-friendly Stroke Expansion](https://linebender.org/gpu-stroke-expansion-paper/) paper has been released. 
+  Follow the link for the outline, the full paper, demos, and a talk at the HPG 2024 conference.
```

Comment:
We should mention something about this demo being created with Xilem Web, and probably have a screenshot of it.